packages/components/src/internal/components/domainproperties/samples/SampleTypePropertiesPanel.tsx

```diff
@@ -386,33 +386,59 @@ class SampleTypePropertiesPanelImpl extends React.PureComponent<
-                                    </>
-                                }
-                            />
-                        </Col>
-                        <Col xs={5}>
-                            <AutoLinkToStudyDropdown
-                                containers={containers}
-                                onChange={this.onFormChange}
-                                autoLinkTarget={ENTITY_FORM_IDS.AUTO_LINK_TARGET}
-                                value={model.autoLinkTargetContainerId}
-                            />
-                        </Col>
-                    </Row>
+                    <>
+                        <Row className="margin-top">
+                            <Col xs={2}>
+                                <DomainFieldLabel
+                                    label="Auto-Link Data to Study"
+                                    helpTipBody={
+                                        <>
+                                            <p>
+                                                Automatically link Sample Type data rows to the specified target study. Only
+                                                rows that include subject and visit/date information will be linked.
+                                            </p>
+                                            <p>
+                                                The user performing the import must have insert permission in the target
+                                                study and the corresponding dataset.
+                                            </p>
+                                        </>
+                                    }
+                                />
+                            </Col>
+                            <Col xs={5}>
+                                <AutoLinkToStudyDropdown
+                                    containers={containers}
+                                    onChange={this.onFormChange}
+                                    autoLinkTarget={ENTITY_FORM_IDS.AUTO_LINK_TARGET}
+                                    value={model.autoLinkTargetContainerId}
+                                />
+                            </Col>
+                        </Row>
+                        <Row className="margin-top">
+                            <Col xs={2}>
+                                <DomainFieldLabel
+                                    label="Linked Dataset Category"
+                                    helpTipBody={
+                                        <>
+                                            Specify the desired category for the Sample Type Dataset that will be created (or appended to) in the
+                                            target study when rows are linked. If the category you specify does not exist, it will be created.
+                                            If the Sample Type Dataset already exists, this setting will not overwrite a previously assigned category.
```

Comment:
minor: could use two `<p>` paragraph tags to separate the help tip text a bit.

Reply:
if you do this here, I suggest doing the same for the assay designer tooltip text.